packages/builder/src/pages/builder/portal/users/_layout.svelte

```diff
@@ -1,12 +1,21 @@
 <script>
-  import { Page } from "@budibase/bbui"
+  import { Page, notifications } from "@budibase/bbui"
+  import { onMount } from "svelte"
   import { SideNav, SideNavItem, Content } from "components/portal/page"
   import { isActive, goto } from "@roxi/routify"
-  import { menu } from "stores/portal"
+  import { menu, features } from "stores/portal"
 
   $: wide = $isActive("./users/index") || $isActive("./groups/index")
   $: pages = $menu.find(x => x.title === "Users")?.subPages || []
   $: !pages.length && $goto("../")
+
+  onMount(async () => {
+    try {
+      await features.init()
+    } catch (error) {
+      notifications.error("Error fetching feature configs")
```

Comment:
Generally a good idea to expose the error as well when notifying - makes debugging anything that goes wrong with users a lot easier (rather than getting them to open network tab etc).

Reply:
I just copied the pattern for the other `notifications.error`. Maybe we should pass the error and to expose it in the helper itself